src/main/java/uk/gov/hmcts/reform/sandl/snlevents/actions/hearingpart/AssignHearingPartToSessionAction.java

```diff
@@ -70,6 +71,10 @@ public void getAndValidateEntities() {
             throw new SnlEventsException("Hearing part can not be listed");
         }
 
+        if (hearingPart.getHearing().isMultiSession()) {
+            throw new SnlEventsException("Hearing part cannot be part of a multi-session!");
```

Comment:
That message is misleading. I'd say "Cannot assign hearing part which is a part of a multi-session"

Reply:
Done! :)